CHANGELOG.md

```diff
@@ -6,14 +6,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [v1.0.0-rc.2] - 2021-03-30
+
 ### Added
```

Comment:
We should've removed the empty chapters here, but we can fix this after the release.

Reply:
Ah, yeah - good catch.